lib/teleport-rb.rb

```diff
@@ -11,13 +11,46 @@ def initialize auth_server:, identity_file:
     @identity_file = identity_file
   end
 
-  def nodes labels={}
+  private
+
+  def execute_command command, handle_json_errors: true
+    stdout, stderr, status = Open3.capture3(command)
+    if handle_json_errors
+      begin
+        MultiJson.load(stdout)
+      rescue MultiJson::ParseError => e
+        raise ResponseError.new(e)
+      end
+    else
+      raise Error.new("Command failed with status #{status} and stderr:\n#{stderr}") unless status == 0
```

Comment:
Would be good to put the command in the error message too.

Reply:
You mean the name of the command that failed?

Reply:
Yeah, the whole command that failed.